test/functional/Jsr292/playlist.xml

```diff
@@ -512,4 +535,4 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 			<impl>ibm</impl>
 		</impls>
 	</test>
-</playlist>
+</playlist>
```

Comment:
Please restore the final newline (every non-empty text file should end with one).